pkg/template/controller/templateinstance_controller.go

```diff
@@ -100,25 +110,35 @@ func (c *TemplateInstanceController) handle(templateInstance *templateapi.Templa
 	return err
 }
 
+func (c *TemplateInstanceController) authorize(u user.Info, action *authorizationapi.Action) error {
+	sar := authorizationapi.AddUserToSAR(u, &authorizationapi.SubjectAccessReview{Action: *action})
```

Comment:
Use the upstream `subjectaccessreviews.authorization.k8s.io`  We're going to (someday) remove our endpoint.